src/main/java/net/dries007/tfc/CommonEventHandler.java

```diff
@@ -71,7 +71,7 @@ public static void onBlockHarvestDrops(BlockEvent.HarvestDropsEvent event)
     /**
      * Handler for {@link IPlaceableItem}
      * To add a new placeable item effect, either implement {@link IPlaceableItem} or see {@link IPlaceableItem.Impl} for vanilla item usages
-     *
+     * <p>
```

Comment:
What's up with all these?

Reply:
Not sure either, that's what IntelliJ did that with CodeStyle.xml

I can clean that for you if you need it.

Reply:
How strange. It doesn't add them for me. 🤷‍♂